packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -1132,6 +1137,91 @@ function commitLayoutEffectOnFiber(
   }
 }
 
+function abortParentMarkerTransitions(
+  deletedFiber: Fiber,
+  nearestMountedAncestor: Fiber,
+  abort: TransitionAbort,
+) {
+  let fiber = deletedFiber;
+  let isInDeletedTree = true;
+  while (fiber !== null) {
+    const instance = deletedFiber.stateNode;
+    switch (fiber.tag) {
+      case TracingMarkerComponent:
+        const transitions = instance.transitions;
+
+        const markerInstance = fiber.stateNode;
+        const markerTransitions = markerInstance.transitions;
+        const abortMarker = Array.from(transitions).some(transition =>
```

Comment:
Why are you converting this to an array? Seems like the only thing you're doing is iterating over it

Reply:
In fact, isn't it an array already? `Array.from` does a copy, which is unnecessary in this case and generally something to try to avoid.

Reply:
It's a set actually but I'm just going to use `forEach` to loop over it